### PR TITLE
data-json - Allow object be iterated for sub-template + additional attr support

### PR DESCRIPTION
**Pull request by duboisp**
This is minor change. 
* It allow object to be iterated for sub-template. This was already possible for template without sub-template.
* data can now be mapped into the following attribute: 
  * `lang` ease the JSON content mapping in pages in different languages
  * `hreflang` ease the JSON content mapping in pages in different languages
  * `aria-*` was documented but not allowed in the code
  * `role` was documented but not allowed in the code
  * `action` To be usable as form submission fallback solution.

Use case: To automate the insertion of component versioning information into each GCWeb component documentation page.

### CODE REVIEW

**Review by Ricokola — DISMISSED** [review]

File: src/plugins/wb-data-json/data-json.js
Comment (GormFrank) [inline src/plugins/wb-data-json/data-json.js:330]: Good job avec cette conditionnelle, love it! This part requires deep knowledge of Javascript for sure:
`!( cached_value instanceof String ) && typeof cached_value === "object"`

File: src/plugins/wb-data-json/demo/data-en.json
Comment (GormFrank) [inline src/plugins/wb-data-json/demo/data-en.json:46]: @duboisp Not a big deal, but you never use the "description" field in your example.

**Review by GormFrank — DISMISSED** [review]
@duboisp Small comment but all good! Merci @Ricokola pour ta revue aussi :)

**Comment by GormFrank** [comment]
Let's wait for the build to be fixed for this one before we merge. May require a rebase.

File: src/plugins/wb-data-json/demo/data-en.json
Comment (duboisp) [reply src/plugins/wb-data-json/demo/data-en.json:46]: Je manquais d'inspiration, on peut toujours le changer si tu as une proposition. :-)

**Comment by duboisp** [comment]
@GormFrank @Ricokola I only rebased and I need you to re-approve. Thanks

**Review by GormFrank — APPROVED** [review]

**Pull request merged**
